frontend/src/modules/automation/components/automation-form.vue

```diff
@@ -284,7 +230,6 @@ const doSubmit = () => {
 };
 
 onMounted(() => {
-  fetchIntegrations();
```

Comment:
Are we now showing all platforms in the automation Matching platform(s) list?

Reply:
why is removed here is just because I put fetching already to automations page, because we needed integrations info already to check if hubspot is connected in order to disable create automation dropdown.